### DISCUSSION TITLE
M-ATV

### DISCUSSION THREAD

**Discussion by swissembedded** [projectchrono/chrono#448] [Ideas]
Hi, looking for a model of the M-ATV Oshkosh vehicle.
Best Dani

**Comment by rserban** [comment]
Dani - we do not have data for this vehicle.  
In general and assuming a vehicle does not have any strange, non-standard components (which doesn't seem to be the case with the M-ATV), obtaining consistent and accurate data is the most difficult step in creating a new model in Chrono::Vehicle.  That is even more difficult for those models that are distributed with Chrono, as the data must be open and freely available for inclusion in a BSD-licensed project.
